src/app/components/ATIAnalytics/canonical/index.tsx

```diff
@@ -9,14 +9,14 @@ import sendPageViewBeaconLite, {
 import sendBeacon from '#app/lib/analyticsUtils/sendBeacon';
 import addInlineScript from '#app/lib/utilities/addInlineScript';
 import { ATIAnalyticsProps } from '../types';
+import trackingPixelUrl from './getNoScriptTrackingPixelUrl';
```

Comment:
Can we rename this and references - we tend to use the same name as the file for the default export

```suggestion
import getNoScriptTrackingPixelUrl from './getNoScriptTrackingPixelUrl';
```